third_party/llvm-project/include/llvm/ObjectYAML/DWARFYAML.h

```diff
@@ -82,6 +82,7 @@ struct Loc {
   uint32_t Start;
   uint32_t End;
   std::vector<uint8_t> Location;
+  uint64_t Offset;
```

Comment:
It should be possible to calculate the Location offset directly in updateLoc(), so these changes are not necessary.

Reply:
In the second commit, there are changes to calculate the Location offset directly in updateLoc(). However the code becomes more convoluted, and I understand that we are already making a number of binaryen-specific changes to the third-party code, so it's probably better to add `Offset` to the `Loc` struct here.